src/app/word-cloud/word-cloud.component.scss

```diff
@@ -0,0 +1,21 @@
+@import '../../_lib/sass/src/variables.scss';
+
+.word-cloud-container {
+    width: 400px;
+    height: 400px;
+    margin-top: 16px;
+    background-color: $level2;
+    border-radius: $border-radius-large;
+
```

Comment:
Suggestion:  Adding these rules highlight how well the resizing works
```
resize: both;
overflow: hidden;
```

Reply:
Good call. I've never considered using the resize property on anything but a textarea